File: mobile/app/src/main/java/ua/nure/makieiev/factory/model/entity/FilterUnit.kt
package ua.nure.makieiev.factory.model.entity

import java.time.LocalDate

data class FilterUnit(
    var id: Long? = null,
    var filter: Filter? = null,
    var unit: Unit? = null,
    var dateInstall: String? = null,
    var status: String? = null,
    var dateRemoval: LocalDate? = null
)

File: frontend/ark-project/src/app/models/entity/FilterUnit.ts
import {Filter} from './Filter';
import {Unit} from './Unit';

export class FilterUnit {

  id: number;
  filter: Filter;
  unit: Unit;
  dateInstall: string;
  status: string;
  dateRemoval: string;

  constructor(id: number, filter: Filter, unit: Unit, dateInstall: string, status: string, dateRemoval: string) {
    this.id = id;
    this.filter = filter;
    this.unit = unit;
    this.dateInstall = dateInstall;
    this.status = status;
    this.dateRemoval = dateRemoval;
  }

}
